### PR TITLE
Re: #913 added filters

### PR DESCRIPTION
**Pull request by aloncohenn**

### CODE REVIEW

File: src/Search/Elastica/QueryFactory.php [hunk 1]
```diff
@@ -103,6 +103,8 @@ protected function applyDateFilters(SearchNodesRequest $request, ParsedQuery $pa
             ['query' => 'created_before', 'field' => 'created_at', 'operator' => ComparisonOperator::LT()],
             ['query' => 'updated_after', 'field' => 'updated_at', 'operator' => ComparisonOperator::GT()],
             ['query' => 'updated_before', 'field' => 'updated_at', 'operator' => ComparisonOperator::LT()],
+            ['query' => 'published_after', 'field' => 'published_at', 'operator' => ComparisonOperator::LT()],
```
Comment (gdbrown) [inline src/Search/Elastica/QueryFactory.php]: @cohencodes comparison operator should be GT for the _after_

**Review by gdbrown — CHANGES_REQUESTED** [review]
@cohencodes update comparison operator for after and add note to changelog (patch)

File: src/Search/Elastica/QueryFactory.php (hunk 1 above)
Comment (aloncohenn) [reply src/Search/Elastica/QueryFactory.php]: @gdbrown fixed

**Review by gdbrown — CHANGES_REQUESTED** [review]
@cohencodes update changelog with ntoes about the change

**Review by gdbrown — APPROVED** [review]

**Pull request merged**